packages/vite/src/node/logger.ts

```diff
@@ -129,7 +129,7 @@ export function printServerUrls(
         const type = detail.address.includes('127.0.0.1')
           ? 'Local:   '
           : 'Network: '
-        const host = detail.address
+        const host = detail.address.replace('127.0.0.1', 'localhost')
```

Comment:
Do we want to use `defaultHostname` here too?

Reply:
This is going to be always 'localhost' when calling it.